cob_default_robot_config/cob3-9/command_gui_buttons.yaml

```diff
@@ -46,15 +46,6 @@ group5:
     [home,move,home],
     [folded,move,folded],
     [pregrasp,move,pregrasp]]
-group6:
```

Comment:
The frontside positions are not defined , look https://github.com/ipa320/cob_robots/blob/indigo_dev/cob_default_robot_config/cob3-9/arm_joint_configurations.yaml and , if you take a look of the history, these positions were never defined. When we delivered cob3-8 and cob3-9 robots, we decided together remove these positions to the default setup of the cob3s.